aea/helpers/multiaddr/base.py

```diff
@@ -102,15 +102,21 @@ def __init__(self, host: str, port: int, public_key: str):
             raise Exception("Malformed public key:{}".format(str(e)))
 
         self._public_key = public_key
-        self._peerid = self._compute_peerid()
+        self._peerid = self.compute_peerid(self._public_key)
 
-    def _compute_peerid(self) -> str:
-        """
-        Compute base58 representation of libp2p PeerID from Bitcoin EC encoded Secp256k1 public key
+    @staticmethod
+    def compute_peerid(public_key: str) -> str:
```

Comment:
a slight refactoring to make `compute_peerid` usable from other parts of the code (in particular, I used it in the `get_multiaddress` implementation).